open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/listeners/AssetLineageOMRSTopicListener.java

```diff
@@ -216,4 +236,22 @@ private void processUpdatedEntityEvent(EntityDetail originalEntity, EntityDetail
     private void processDeletedEntity(EntityDetail entity) {
     }
 
+
+    private void processSemanticAssignmentDeletion(Relationship relationship, String serviceOperationName)  {
+        DeletePurgedRelationshipEvent deletionEvent = new DeletePurgedRelationshipEvent();
+        try {
+
+            GlossaryHandler glossaryHandler = instanceHandler.getGlossaryHandler(serverUserName, serverName, serviceOperationName);
+            GlossaryTerm glossaryTerm = glossaryHandler.getGlossaryTerm(relationship.getEntityTwoProxy(), serverUserName);
+
+            deletionEvent.setGlossaryTerm(glossaryTerm);
+            deletionEvent.setEntityGuid(relationship.getEntityOneProxy().getGUID());
+            deletionEvent.setEntityTypeDef(relationship.getEntityOneProxy().getType().getTypeDefName());
+            deletionEvent.setOmrsInstanceEventType(OMRSInstanceEventType.DELETE_PURGED_RELATIONSHIP_EVENT);
+
+            publisher.publishRelationshipEvent(deletionEvent);
+        } catch (Exception e) {
+            log.error(e.getMessage());
```

Comment:
catch Exception in general is not a good practice: swallow all the exceptions, including ones that might be handled in an upper level but they never reach it.

Also error message of log suppose to include some details that makes it easy to identify. Something like:
`log.error("Error process semantic assignment deletion for "+ relationship.getEntityOneProxy().getGUID() + ". Exception message is:  "+  e.getMessage());`